src/main.cpp

```diff
@@ -944,9 +939,43 @@ int main(int argc, char *argv[]) {
 #if defined(USENVML)
   if (nvidia_log_power || nvidia_log_temp)
   {
-    cout << "Using NVML interface..." << endl;
-    h5_create_dir(out_name, "/housekeeping");
-    h5_create_dir(out_name, "/housekeeping/nvidia");
+      //Get Program Files path from system
+      TCHAR pf[MAX_PATH];
```

Comment:
This needs additional include guards and/or definitions on Linux:
```
toolkitICL/src/main.cpp:943:7: error: ‘TCHAR’ was not declared in this scope
       TCHAR pf[MAX_PATH];
       ^~~~~
/home/hendrik/Research/AIKEF/toolkitICL/src/main.cpp:944:32: error: ‘pf’ was not declared in this scope
       SHGetSpecialFolderPath(0,pf,CSIDL_PROGRAM_FILES,FALSE);
                                ^~
/home/hendrik/Research/AIKEF/toolkitICL/src/main.cpp:944:35: error: ‘CSIDL_PROGRAM_FILES’ was not declared in this scope
       SHGetSpecialFolderPath(0,pf,CSIDL_PROGRAM_FILES,FALSE);
                                   ^~~~~~~~~~~~~~~~~~~
[ 67%] Linking CXX executable copy_char
/home/hendrik/Research/AIKEF/toolkitICL/src/main.cpp:944:35: note: suggested alternative: ‘CL_PROGRAM_IL’
       SHGetSpecialFolderPath(0,pf,CSIDL_PROGRAM_FILES,FALSE);
                                   ^~~~~~~~~~~~~~~~~~~
                                   CL_PROGRAM_IL
/home/hendrik/Research/AIKEF/toolkitICL/src/main.cpp:944:55: error: ‘FALSE’ was not declared in this scope
       SHGetSpecialFolderPath(0,pf,CSIDL_PROGRAM_FILES,FALSE);
                                                       ^~~~~
/home/hendrik/Research/AIKEF/toolkitICL/src/main.cpp:944:55: note: suggested alternative: ‘FILE’
       SHGetSpecialFolderPath(0,pf,CSIDL_PROGRAM_FILES,FALSE);
                                                       ^~~~~
                                                       FILE
/home/hendrik/Research/AIKEF/toolkitICL/src/main.cpp:944:7: error: ‘SHGetSpecialFolderPath’ was not declared in this scope
       SHGetSpecialFolderPath(0,pf,CSIDL_PROGRAM_FILES,FALSE);
       ^~~~~~~~~~~~~~~~~~~~~~
/home/hendrik/Research/AIKEF/toolkitICL/src/main.cpp:949:11: error: ‘LoadLibraryEx’ was not declared in this scope
           LoadLibraryEx(nvsmi_path.c_str(), NULL, 0);
           ^~~~~~~~~~~~~
/home/hendrik/Research/AIKEF/toolkitICL/src/main.cpp:952:15: error: ‘LoadLibraryEx’ was not declared in this scope
               LoadLibraryEx("nvml.dll", NULL, 0);

```

Reply:
Somehow an `#if defined(_WIN32)` got lost. This should be fixed now.